derive_stringset/src/lib.rs

```diff
@@ -70,3 +79,41 @@ pub fn derive(input: TokenStream) -> TokenStream {
 
     TokenStream::from(expanded)
 }
+
+fn derive_enum(input: &syn::DeriveInput) -> TokenStream {
+    let variants = if let syn::Data::Enum(syn::DataEnum { ref variants, .. }) = input.data {
+        variants
+    } else {
+        // We should not have called derive_enum() if input.data wasn't an enum
+        unreachable!();
+    };
+
+    // Only support simple enums, check each field
+    for v in variants.iter() {
+        match v.fields {
+            syn::Fields::Named(_) | syn::Fields::Unnamed(_) => {
+                unimplemented!("only simple enums are supported")
```

Comment:
Clean up these comments to use proper punctuation. Any panics in this code will be displayed prominently by the compiler to the user:

```
ryan@Gabriel:~/repositories/quartiq/stabilizer$ cargo build                                              
   Compiling stabilizer v0.4.1 (/home/ryan/repositories/quartiq/stabilizer)                              
error: proc-macro derive panicked                                                                        
 --> src/hardware/afe.rs:7:72                                                                            
  |                                                                                                      
7 | #[derive(Copy, Clone, Debug, Serialize, Deserialize, IntoEnumIterator, StringSet)]                   
  |                                                                        ^^^^^^^^^                     
  |                                                                                                      
  = help: message: not implemented  
```